pkg/network/config.go

```diff
@@ -20,10 +19,6 @@ const (
 	DefaultPodmanDomainName = "dns.podman"
 )
 
-var (
-	ErrNetworkNotFound = errors.New("network not found")
```

Comment:
This should have been left and just a redirect to define.ErrNetowrkNotFound, otherwise you are changing the API.